### PR TITLE
BRIDGE REVAMP TRY NUMBER 3!!! [MERGE CONFLICTS tm]

### PR DESCRIPTION
**Pull request by Kmc2000**
If at first you don't succeed...

At this rate just look at this one 😂 

https://github.com/FTL13/FTL13/pull/649

There video xd

https://drive.google.com/file/d/0B9VUbibptgF2MlZuWGlaalc4QlE/view?usp=sharing

### CODE REVIEW

**Comment by IlyaIU** [comment]
ERROR: /obj/item/clothing/head/collectable/HoP - undefined var

**Comment by Kmc2000** [comment]
I'm so fucking confused what the fuck is going on help

**Comment by Kmc2000** [comment]
MERGE CONFLICTS KILLED


🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 
👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌 👌

File: _maps/map_files/Aetherwhisp/Aetherwhisp.dmm [hunk 1]
```diff
@@ -732,7 +732,7 @@
 /area/shuttle/ftl/engine/engineering)
 "bup" = (
 /obj/machinery/power/smes{
-	charge = 5e5
+	charge = 500000
```
Comment (ninjanomnom) [inline _maps/map_files/Aetherwhisp/Aetherwhisp.dmm]: Why are you changing the smes charge in a sprite change PR?

Comment (Kmc2000) [inline _maps/map_files/Aetherwhisp/Aetherwhisp.dmm]: I didn't...

Comment (Kmc2000) [inline _maps/map_files/Aetherwhisp/Aetherwhisp.dmm]: *also 5E5 is 50000 anyway, i think mapmerge did that or something?

**Comment by ike709** [comment]
I don't really like any of the new sprites, honestly. And in the video it looks like you undid the changes in https://github.com/FTL13/FTL13/pull/638

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -1604,6 +1609,47 @@
 	                /obj/item/toy/redbutton)
 	crate_name = "toy crate"
 
+/datum/supply_pack/misc/trekuniforms
+	name = "Outdated Uniforms"
+	cost = 400
```
Comment (ninjanomnom) [inline code/modules/cargo/packs.dm]: Empty crates are worth more than 400, price needs to be at least 700 according to the file comments or cargo can buy this to get free points.

**Comment by Kmc2000** [comment]
@ike709

whoops

**Comment by Kmc2000** [comment]
@ike709

Ok i just checked this, this was due to some weird stuff going on with the maps, this has since been fixed and is running off the latest maps with changes

**Comment by ninjanomnom** [comment]
You really should close this, update your fork and redo your changes. There's a ton of things reverted in this. Trailblazer for example has 2,840 line changes? I think you also revert part of the gas miner.

**Comment by Kmc2000** [comment]
Most of the line changes are from mapmerging though?

**Comment by ninjanomnom** [comment]
There shouldn't be that many line changes from replacing a few objects.

File: ftl13.dme [hunk 3]
```diff
@@ -972,7 +972,6 @@
 #include "code\modules\atmospherics\machinery\components\unary_devices\vent_pump.dm"
 #include "code\modules\atmospherics\machinery\components\unary_devices\vent_scrubber.dm"
 #include "code\modules\atmospherics\machinery\other\meter.dm"
-#include "code\modules\atmospherics\machinery\other\miner.dm"
```
Comment (ninjanomnom) [inline ftl13.dme]: This is for the gas miner iirc

File: _maps/map_files/Trailblazer/Trailblazer.dmm [hunk 4]
```diff
@@ -602,7 +603,14 @@
...
 "bnP" = (
-/obj/machinery/atmospherics/pipe/manifold/supply/hidden,
+/obj/structure/cable{
+	d1 = 4;
+	d2 = 8;
+	icon_state = "4-8"
+	},
+/obj/machinery/atmospherics/pipe/manifold/supply/hidden{
+	dir = 4
+	},
```
Comment (ninjanomnom) [inline _maps/map_files/Trailblazer/Trailblazer.dmm]: Stuff like this shouldn't be changing from just replacing some objects

**Comment by Vivalas** [comment]
No merge conflicts but travis don't want none.

**Comment by ike709** [comment]
@Vivalas I think there's serious issues with this PR...

**Pull request closed without merging**